opentimelineview/timeline_widget.py

```diff
@@ -205,6 +214,67 @@ def __init__(self, *args, **kwargs):
         self.source_name_label.setText('GAP')
 
 
+class Effect(QtWidgets.QGraphicsRectItem):
```

Comment:
what if this was called `EffectsBadge` or `EffectsWidget` or something else like that to disambiguate it from `otio...Effect`

Reply:
what about EffecfItem ? The other pysideWidget are like ClipItem, GapItem, TransitionItem , NestedItem.

Reply:
Sounds great to me!